api/go.mod

```diff
@@ -1,12 +1,12 @@
 module github.com/openstack-k8s-operators/heat-operator/api
 
-go 1.20
+go 1.21
 
 require (
 	github.com/openstack-k8s-operators/lib-common/modules/common v0.4.1-0.20240905123813-174296c09ec6
-	k8s.io/api v0.28.13
-	k8s.io/apimachinery v0.28.13
-	sigs.k8s.io/controller-runtime v0.16.6
+	k8s.io/api v0.29.2
+	k8s.io/apimachinery v0.29.2
+	sigs.k8s.io/controller-runtime v0.17.5
```

Comment:
Should this be v0.17.6 like here https://github.com/openstack-k8s-operators/openstack-operator/pull/1075/files#diff-1a135474df0925a580fa1c89d8925690426c80f754af7142363cb87b6d99ac70R41

Reply:
I just copied this one:
https://github.com/openstack-k8s-operators/keystone-operator/pull/470/files#diff-ba0784a950902285760f839a4ad720a381b2981e8a67147f5f43f7508e6343edR16

But it would be nice to do the latest 0.17.x everywhere.
https://github.com/kubernetes-sigs/controller-runtime/releases/tag/v0.17.6